ui/app/clinical/common/services/visitTabConfig.js

```diff
@@ -41,6 +41,12 @@ angular.module('bahmni.clinical')
                 });
                 firstTabWithDefaultSection.sections = _.unionWith(_.values(mandatorySections), _.values(firstTabWithDefaultSection.sections), _.isEqual);
 
+                var tabWithDefaultSectionsExcludes = _.find(tabs, function (tab) { return tab.defaultSectionsExcludes; });
+                var excludedDefaultSections = tabWithDefaultSectionsExcludes ? tabWithDefaultSectionsExcludes.defaultSectionsExcludes : [];
```

Comment:
the property "defaultSectionExcludes" can be undefined or set to null, right? can we put a check against that. Or does the underscrore library ensure that?

Reply:
Yes, it'll be handled even it if is undefined or null. It'll display all the default sections if the config is not defined, set to null or empty